### PR TITLE
Add message and default route.

### PR DESCRIPTION
**Pull request by afaur**

### CODE REVIEW

File: bin/serve [hunk 1]
```diff
@@ -6,4 +6,4 @@ echo ${BASH_SOURCE[0]}
 
 DIR=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
 
-coffee "$DIR/../src/index.coffee"
+"${DIR}/../node_modules/coffee-script/bin/coffee" "$DIR/../src/index.coffee"
```
Comment (bayleedev) [inline bin/serve:9]: So now I don't need to install coffee myself?

File: package.json [hunk 2]
```diff
@@ -12,6 +12,7 @@
     "js-yaml": "~3.2.5",
     "jade": "~1.8.2",
     "node-static": "~0.7.6",
+    "coffee-script": "*",
```
Comment (bayleedev) [inline package.json:15]: Should this be a specific version?

File: src/index.coffee [hunk 3]
```diff
@@ -90,6 +90,12 @@ class WebServer
   constructor: (port, handler) ->
     @_server = http.createServer(handler)
     @_server.listen(port)
+    # Add status message
+    console.log "------------- Coffee Webserver -------------"
+    console.log "Started server."
+    console.log "listening on port: #{port}."
```
Comment (bayleedev) [inline src/index.coffee:96]: Clever.

File: src/index.coffee [hunk 4]
```diff
@@ -159,6 +165,7 @@ class WebRequest
   # Move this out into Router class and use a routes.yml for mapping
   routes:
     'GET':
+      '/': {"controller": 'FooController', "method": 'index'}
```
Comment (bayleedev) [inline src/index.coffee:168]: :+1:

**Comment by bayleedev** [comment]
What about a `npm start` that calls that bin file?

**Comment by afaur** [comment]
An npm start task would be good.  Your more then welcome to open a PR and add that.  Also if you get a chance to test this on that latest version of node and it ends up working okay, would you mind adding a greater then indicator to the v0.10.33 inside the package.json for engines, node versions supported.

File: package.json (hunk 2 above)
Comment (afaur) [inline package.json:15]: Probably so, if you have a moment could you update that?

**Pull request merged**